### PR TITLE
Add setting for date display format

### LINKED ISSUE #194: Add setting for display of datetime format for the UI
## Problem Definition

Currently in the application there is a hardcoded format for each different place a date is shown. It would be rather nice if a new setting is added to specify the way dates should be displayed. Also if it should be displayed in UTC or in the user system's local time. For a suggested format we could go with the python [strftime](https://docs.python.org/3.6/library/datetime.html#strftime-and-strptime-behavior) but perhaps that would be a lot of work to implement. @kelsos is there a javascript way to provide a unix timestamp and a string format and get a date string? Google searches seem to point to a `moment.js` package but it seems crazy to me to need an external package for something so simple.

## Task

- Add a new setting. `ui_date_display_format`. Default value should be `%d/%m/%Y %H:%M:%S %Z `. Depending on the format we choose
- Use that format in the front-end to display timestamps to date strings. The function where this happens is `timestamp_to_date()` seen [here](https://github.com/rotkehlchenio/rotkehlchen/blob/3bd8ac193db26da73fe48768db036c25ecd2a4ff/ui/utils.ts#L23).

### PR DESCRIPTION
**Pull request by kelsos**
Closes #194

### CODE REVIEW

**Comment by kelsos** [comment]
It seems I have to do some timezone related tests to make sure that everything works properly.

**Comment by kelsos** [comment]
@LefterisJP can you review this PR?

File: rotkehlchen/data_handler.py [hunk 1]
```diff
@@ -75,6 +75,7 @@
     'balance_save_frequency',
     'anonymized_logs',
     'include_gas_costs',
+    'date_display_format',
```
Comment (LefterisJP) [inline rotkehlchen/data_handler.py]: Hmm more changes are needed in the backend. You need to be able to return a default display format from the DB even if there is no value in there (for all people who have not yet upgraded to the changes of this PR).

Check here for how to achieve this: 

https://github.com/rotkehlchenio/rotkehlchen/blob/master/rotkehlchen/db/dbhandler.py#L391-L433

And also change the tests here:

https://github.com/rotkehlchenio/rotkehlchen/blob/dbdee73c98bc55940069d911eeebb65ed9825f1a/rotkehlchen/tests/test_db.py#L143

**Comment by kelsos** [comment]
@LefterisJP Can you re-review the PR? or at least the latest changes? There was something weird happening with the CI yesterday but now tests seem to pass.

**Pull request merged**